api/src/public/login/script.js

```diff
@@ -68,6 +68,10 @@ const requestSSOLogin = function(e){
 };
 
 const requestTokenLogin = (retry = 20) => {
+  if (!isUsingSupportedBrowser()) {
```

Comment:
I don't think this is what we want, because this will block any unsupported browser from using token login. Also this does _not_ avoid Safari (since that function is separate). 

Can you please look into moving the call to `checkUnsupportedBrowser` to before the token login check and have the Safari check disable token login somehow?
Even when calling the token login link, a user on Safari should see the Safari "unsupported browser" message.